app/javascript/pages/about/section-partners/section-partners.js

```diff
@@ -334,6 +334,11 @@ const partnersCollaborators = [
     img: '/assets/logos/whrchover.png',
     link: 'http://whrc.org/',
     title: 'Woods Hole Research Center'
+  },
+  {
+    img: '/assets/logos/evidensiahover.png',
```

Comment:
We display all logos in alphabetical order to avoid preference. Could you move it to the right spot?

Reply:
Ah I didn't noticed. Fixed 👍